heron/healthmgr/src/java/org/apache/heron/healthmgr/common/PhysicalPlanProvider.java

```diff
@@ -78,11 +78,9 @@ public synchronized void onEvent(ContainerRestart event) {
 
   @Override
   public synchronized PhysicalPlan get() {
+    physicalPlan = stateManagerAdaptor.getPhysicalPlan(topologyName);
```

Comment:
hmm. this seems to change some assumption that physicalplan doesn't change by itself. is this really necessary to get physical plan every time? and how often the function get called?

Reply:
The pplan in healthmgr does no change itself unless resolver does action and triggers pplan to change, which is healthmgr internal motivation.

Healthmgr does not receive external pplan change notice due to its polling model. if healthmgr does not fetch the latest pplan, it does not know whether pplan changes.

The frequency depends on policy cycle. typical value is 2 min.

Reply:
kk. thx.